author/basics/workflow.adoc

```diff
@@ -125,7 +125,7 @@ your document in their preferred channels, such as online or print.
 
 === Related links
 
-* link:/author/topics/creating_new_document[Creating a new document from a template]
+* link:/author/ieee/authoring-guide/new-doc-template[Creating a new document from a template]
```

Comment:
This is wrong, the basic workflow should *not* point to IEEE.

Reply:
Fixed